openmdao/devtools/xdsm_viewer/xdsm_writer.py

```diff
@@ -518,6 +522,14 @@ def number_label(number, text, alignment):
 
     if writer_name == 'pyxdsm':  # pyXDSM
         x = XDSMWriter()
+
```

Comment:
I am leaning against doing the following here... I think it's better handled in pyXDSM and adding this logic would put the burden on us to add some awkward tests... thoughts?

Reply:
This does seem like pyXDSM's responsibility.

Reply:
I also think it's better handled by pyXDSM.

Reply:
removed